scripts/tools/linux-install-githubctl

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [[ "$(uname)" != "Linux" ]]; then
+  echo "Run on Linux only."
+  exit 1
+fi
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+. ${DIR}/all-utilities || { echo "Cannot load Bash utilities" ; exit 1 ; }
+
+
+# Install githubctl
+function install_githubctl() {
+    pushd $DIR/..
+    git clone https://github.com/istio/test-infra.git -b master --depth 1
```

Comment:
you are already in test-infra

Reply:
no the docker image just has the scripts directory added. test-infra itself is not checked out